cmd/http/main.go

```diff
@@ -41,5 +47,36 @@ func main() {
 	router.POST("/v1/generateSBOM", controller.GenerateSBOM)
 	router.POST("/v1/scanImage", controller.ScanCVE)
 
-	logger.L().Ctx(ctx).Fatal("router error", helpers.Error(router.Run()))
+	srv := &http.Server{
+		Addr:    ":8080",
```

Comment:
I think it's better to make the port configurable

Reply:
Indeed, this is part of the industrialization. Before it was defaulted to 8080, I had to make it explicit when declaring the server.
Please focus on the graceful shutdown sequence that is required by the queue you asked for :)